src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -104,7 +104,7 @@ async function initializeConnection(options: CircuitStartOptions, logger: Logger
     return connection.send('OnLocationChanged', uri, intercepted);
   });
 
-  connection.on('JS.AttachComponent', (componentId, selector) => attachRootComponentToLogicalElement(0, circuit.resolveElement(selector), componentId));
+  connection.on('JS.AttachComponent', (componentId, selector) => attachRootComponentToLogicalElement(0, circuit.resolveElement(selector), componentId, false));
```

Comment:
Not sure if I'm missing something here or if this is an omission, but how come we can always pass `false` here? Don't we need to receive the "append" flag from the .NET side and pass it through?

Reply:
I may be missing something, but the `appendContent` option only applies for WebAssembly, right? No one would never need to do this on Blazor server since it doesn't attach root components to existing DOM elements in the same way WebAssembly does?

Reply:
I see - got it! Makes sense. I guess the question goes away anyway if we're using pseudoselectors instead of the flag.